[2022-07-29 11:43:46,144] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2022-04-03T00:00:00+00:00 [queued]>
[2022-07-29 11:43:46,164] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2022-04-03T00:00:00+00:00 [queued]>
[2022-07-29 11:43:46,165] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:43:46,165] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 11:43:46,165] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:43:46,187] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_name> on 2022-04-03 00:00:00+00:00
[2022-07-29 11:43:46,192] {standard_task_runner.py:52} INFO - Started process 6881 to run task
[2022-07-29 11:43:46,196] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator', 'get_name', 'scheduled__2022-04-03T00:00:00+00:00', '--job-id', '944', '--raw', '--subdir', 'DAGS_FOLDER/CreateDagWithPythonOperator.py', '--cfg-path', '/tmp/tmp4hxly_il', '--error-file', '/tmp/tmpieof9bgd']
[2022-07-29 11:43:46,197] {standard_task_runner.py:80} INFO - Job 944: Subtask get_name
[2022-07-29 11:43:46,296] {task_command.py:369} INFO - Running <TaskInstance: dag_with_python_operator.get_name scheduled__2022-04-03T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 11:43:46,422] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=dag_with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-04-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-03T00:00:00+00:00
[2022-07-29 11:43:46,463] {python.py:173} INFO - Done. Returned value was: None
[2022-07-29 11:43:46,485] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator, task_id=get_name, execution_date=20220403T000000, start_date=20220729T114346, end_date=20220729T114346
[2022-07-29 11:43:46,545] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 11:43:46,639] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
